### PR TITLE
Add test for notebook rendering

### PR DESCRIPTION
**Pull request by audy**
- Add test for notebook rendering w/ diff using `diff-pdf`
- Add Github test workflow
- Fix Weasyprint transparency issue

### CODE REVIEW

File: test/test_render_notebook.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    "stderr:",
+                    result.stderr.decode("utf-8"),
+                ]
+            )
+        )
+
+    return result
+
+
+def test_render_notebook(tmp_path):
```
Comment (alenajk) [inline test/test_render_notebook.py:40]: Maybe I'm missing something - where is `tmp_path` coming from?

File: test/test_render_notebook.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    run_docker_container(
+        tmp_path, ["nbconvert", "--execute", "--to", "onecodex_pdf", "/share/example.ipynb"]
```
Comment (alenajk) [inline test/test_render_notebook.py:42]: nit: Could you add a short comment about the existence of `onecodex_pdf` as a format?

File: test/test_render_notebook.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import pytest
+import os
+import subprocess
+
+IMAGE = "docker-onecodex-notebook"
+
+
+def run_docker_container(tmp_path, container_command):
```
Comment (alenajk) [inline test/test_render_notebook.py:8]: `tmp_path` isn't used in this function?

File: test/test_render_notebook.py (hunk 1 above)
Comment (audy) [reply test/test_render_notebook.py:40]: This is a [pytest fixture](https://docs.pytest.org/en/latest/how-to/tmp_path.html)

File: test/test_render_notebook.py (hunk 3 above)
Comment (audy) [reply test/test_render_notebook.py]: That was leftover. Removed.

File: test/test_render_notebook.py (hunk 2 above)
Comment (audy) [reply test/test_render_notebook.py]: Done.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          pip install --progress-bar=off pytest
+
+      - name: Install diff-pdf
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends \
+            libpoppler-glib-dev \
+            libwxgtk3.0-gtk3-dev
+
+          git clone --depth=1 --branch master https://github.com/vslavik/diff-pdf.git
```
Comment (petraszd) [inline .github/workflows/test.yml:33]: Is it not available on Ubuntu `apt-get`?

Comment (petraszd) [reply .github/workflows/test.yml:33]: Hmmm... It seems that it is not. It seems that it is developed by Fedora loving Debian hating czech.

Comment (petraszd) [reply .github/workflows/test.yml:33]: Can it check out not the latest master but some concrete tag (`v0.5.1`)?

Though I am not sure if that would be more secure and you can't override tags if some malicious actor would take control of `diff-pdf` repo.

**Review by petraszd — APPROVED** [review]

File: .github/workflows/test.yml (hunk 4 above)
Comment (audy) [reply .github/workflows/test.yml]: @petraszd good idea. pinned to v0.5.1

**Pull request merged**